plugins/inputs/intel_powerstat/intel_powerstat_sample_config.go

```diff
@@ -1,11 +0,0 @@
-//go:build linux
```

Comment:
Please re-add this file. This is how we automatically generate the sample config in code.